src/components/Shop/ShopWindow/ShopItem.tsx

```diff
@@ -1,24 +1,35 @@
 import React, { FC } from "react";
 import styled from "styled-components";
+import { Product } from "types/inventory";
+import { IMAGE_URI } from "../../../api";
+
+type ShopWindowItemProps = {
+  storeObject: Product;
```

Comment:
StoreObject is a poor name, it does not really describe the variable. Maybe rename it to just product or item or something else?